packages/dashboard/src/components/widgets/widget.tsx

```diff
@@ -1,19 +1,29 @@
 import { SiteWiseQuery } from '@iot-app-kit/source-iotsitewise';
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useDrop } from 'react-dnd';
 import { DashboardMessages } from '../../messages';
 
 import { DashboardConfiguration, Widget } from '../../types';
 import { gestureable } from '../internalDashboard/determineTargetGestures';
 import DynamicWidgetComponent from './dynamicWidget';
+import { ItemTypes } from '../dragLayer/itemTypes';
 
 import './widget.css';
 
+export type AssetsParam = {
+  assetId: string;
+  properties: {
+    propertyId: string;
+  }[];
+};
+
 export type WidgetProps = {
   readonly: boolean;
   query?: SiteWiseQuery;
   isSelected: boolean;
   cellSize: number;
   widget: Widget;
+  widgets: Widget[];
```

Comment:
Is this used in the component?

Reply:
Oops, it was at one point in development but is no longer. Removed.